<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaneLabelingTool</class>
 <widget class="QWidget" name="LaneLabelingTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>LaneLabelingTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(210, 210, 210);
font: 700 9pt &quot;Microsoft YaHei&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,50,0,0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="videoMuneLayout" stretch="0,0,40,0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="myButton" name="openButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::FolderNew"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1_1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>文件名: 无</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="editStatusKey">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>编辑状态:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="editStatusValue">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>无</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="myButton" name="getTimePosButton">
         <property name="styleSheet">
          <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
         </property>
         <property name="text">
          <string>截取时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QVideoWidget" name="videoWidget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="videoMsgLayout" stretch="0,20,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>00:00:00.000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="playCtrSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="videoFullTime">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>00:00:00.000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="frameNum">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>帧:0/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,1,0,0,1,0,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="firstTureTimeKey">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>首帧/时间:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstFrameValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="trueStartTimeValue">
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lastTrueTimeKey">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>尾帧/时间:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lastFrameValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="trueEndTimeValue">
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="laneCountKey">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>车道数:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="laneCountValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="laneNumKey">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>所在车道:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="laneNumValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px; 
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0;
	selection-color: #F2F2F2; 
	font-size: 10pt; </string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,8">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="myButton" name="prevFrameButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="myButton" name="playPauseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="myButton" name="nextFrameButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::GoNext"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="myButton" name="stopButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="frameBegineButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>设置首帧</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="frameEndButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>设置尾帧</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="eidtButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>修  改</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="deleteButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>删  除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>取  消</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="myButton" name="saveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top:2px solid  #6B6B6B;
background-color: rgb(51, 51, 51);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>保  存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myButton</class>
   <extends>QPushButton</extends>
   <header location="global">myButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/LaneLabelingTool.qrc"/>
 </resources>
 <connections/>
</ui>
